drivers/bluetooth/hci/hci_da1469x.c

```diff
@@ -0,0 +1,497 @@
+/* hci_da1469x.c - DA1469x CMAC IPC Bluetooth driver */
+
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/drivers/bluetooth/hci_driver.h>
+#include <zephyr/irq.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/random/random.h>
+
+#include <common/bt_str.h>
+
+#include <DA1469xAB.h>
+#include <mbox.h>
+#include <shm.h>
+#include <rand.h>
+
+#define LOG_LEVEL CONFIG_BT_HCI_DRIVER_LOG_LEVEL
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(hci_da1469x);
+
+#define H4_NONE 0x00
+#define H4_CMD  0x01
+#define H4_ACL  0x02
+#define H4_EVT  0x04
+#define H4_ISO  0x05
```

Comment:
These are all available in `include/zephyr/bluetooth/hci_types.h` please use those defines instead.